package.json

```diff
@@ -98,9 +97,21 @@
     "react-virtualized-select": "^3.1.3"
   },
   "devDependencies": {
+    "@types/classnames": "^2.2.6",
+    "@types/enzyme": "^3.1.13",
+    "@types/enzyme-adapter-react-16": "^1.0.3",
+    "@types/extract-text-webpack-plugin": "^3.0.3",
+    "@types/leaflet": "^1.2.9",
+    "@types/react-fontawesome": "^1.6.4",
+    "@types/react-leaflet": "^1.1.6",
+    "@types/react-transition-group": "^2.0.13",
+    "@types/react-virtualized-select": "^3.0.7",
+    "@types/webpack-bundle-analyzer": "^2.9.2",
+    "@types/webpack-node-externals": "^1.6.3",
```

Comment:
These should only be in the dedicated PR.

Reply:
Yep, you're right I'll remove that